cni/pkg/install-cni/cmd/root.go

```diff
@@ -39,6 +39,7 @@ var rootCmd = &cobra.Command{
 		if cfg, err = constructConfig(); err != nil {
```

Comment:
doesn't need to be this PR, but one thing I noticed is its hard to tell what is an Istio CNI log in kubelet. Maybe we should have them all in `istiocni` scope or something?

Reply:
Yeah that is also something I am planning to do, to have some label/scope for easy filtering.